pkg/skaffold/event/v2/cloudrun.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"fmt"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
+	proto "github.com/GoogleContainerTools/skaffold/proto/v2"
+)
+
+func CloudRunServiceReady(r, url, revision string) {
```

Comment:
nit: Can you add checking for the event the be emitted as expected as part of the integration testing in:
https://github.com/GoogleContainerTools/skaffold/blob/main/integration/deploy_cloudrun_test.go